### PR TITLE
Enable completions for TypeScript.

### PR DESCRIPTION
**Pull request by phyllisstein**
👋 Hey folks! Made a tiny tweak to your handy package to enable the completion search in TypeScript, and figured I may as well kick it back over to you. Thanks for all your work on this!

### CODE REVIEW

**Comment by Haroenv** [comment]
This seems like a great change! Thanks

File: lib/suggestions-provider.js [hunk 1]
```diff
@@ -10,8 +10,8 @@ export default class {
   constructor(packagesIndex) {
     this.packagesIndex = packagesIndex;
     // This will work on JavaScript and CoffeeScript files, but not in js comments.
-    this.selector = '.source.js';
-    this.disableForSelector = '.source.js .comment';
+    this.selector = '.source.js, .source.ts, .source.tsx';
+    this.disableForSelector = '.source.js .comment, .source.ts .comment, .source.tsx .comment';
```
Comment (Haroenv) [inline lib/suggestions-provider.js]: Can you run `yarn eslint . --fix`?

Comment (phyllisstein) [reply lib/suggestions-provider.js]: Done and done.

**Comment by rayrutjes** [comment]
Thanks a ton @phyllisstein .

I've rebased to follow conventional commit convention and merged your commits here: #24 🎉 

Is now released under V0.2.0

**Pull request closed without merging**